packages/rum-core/src/common/config-service.js

```diff
@@ -240,15 +241,19 @@ class Config {
   }
 
   getLocalConfig() {
-    let config = sessionStorage.getItem(LOCAL_CONFIG_KEY)
+    let config = localStorage.getItem(LOCAL_CONFIG_KEY)
```

Comment:
This is going to affect centralConfig as well right? It's a breaking change as session is different from local storage and if a user configured transactionSampleRate, then thats not going to change ever unless the user explicits clears the storage.

Reply:
Yeah, I agree it should be a breaking change. But since the session feature is behind a feature flag (disabled by default), we can use that flag here to use the correct storage and only switch on our upcoming major version.

Reply:
I've changed added the flag here to use the correct storage until we're ready to make the change in a major version.